java/ql/test/library-tests/dataflow/taintgettersetter/taintgettersetter.ql

```diff
@@ -0,0 +1,25 @@
+import java
```

Comment:
Up to you, but I prefer to make such queries `path-problems` so I can easily inspect the data flow paths in the IDE.

Reply:
I wanted to keep the `.expected` file more readable, so I prefer to keep it as-is.